precompiles/xcm/src/lib.rs

```diff
@@ -22,22 +22,29 @@ use fp_evm::{PrecompileHandle, PrecompileOutput};
 use frame_support::{
     dispatch::{Dispatchable, GetDispatchInfo, PostDispatchInfo},
     pallet_prelude::Weight,
-    traits::Get,
+    traits::{ConstU32, Get},
 };
+pub const XCM_SIZE_LIMIT: u32 = 2u32.pow(16);
+type GetXcmSizeLimit = ConstU32<XCM_SIZE_LIMIT>;
```

Comment:
Should this be under primitives, xcm constants?

Reply:
you are right, I have aded it now.